notebooks/data-sources/TestGrid/metrics/correlated_failures.ipynb

```diff
@@ -4,40 +4,52 @@
    "cell_type": "markdown",
```

Comment:
<p>Small nit, but I think doing it like this might be more efficient</p><pre class="ql-syntax" spellcheck="false">vocab = set()
count = 0
for fg in failure_groups:
&nbsp; &nbsp; count += len(fg)
&nbsp; &nbsp; vocab.update(fg)

print(count)
len(vocab)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Much better : ) thanks</p>